pages/common/mvn.md

```diff
@@ -0,0 +1,23 @@
+# mvn
+
+> Apache Maven. A build automation tool used for JVM environment projects.
+
+- Most common usage is to invoke a life cicle phase:
```

Comment:
typo at cycle.

This description can be improved. No need to mention "Most common usage". It does not say anything about what `mvn package` does. Can you expand a bit on what is meant by a life cycle phase ?

Reply:
It is a little bit difficult to add a phase definition without the maven context, what do you think about this:
"Invoke a Lifecycle phase (a set of steps for the whole project Lifecycle)"?

Reply:
we can live with it.